### DISCUSSION TITLE
Don't repaet quotes

### DISCUSSION THREAD

**Discussion by Antosser** [monkeytypegame/monkeytype#3504] [Ideas] [answered]
## Idea
Today, I was playing a lot of monkeytype in quote mode. I saw a quote that I clearly remember I did before.

## How to implement
You could save all the completed quotes somewhere and if a quote is on that list you won't type it again. If every quote is in the list, the list gets cleared.

**Comment by Miodec** [comment, marked as answer]
> You could save all the completed quotes somewhere and if a quote is on that list you won't type it again. If every quote is in the list, the list gets cleared.

That's exactly how it works already. Some languages / quote lengths just don't have that many entries unfortunately.
